src/roswire/ros2/launch.py

```diff
@@ -33,33 +33,11 @@ def for_app_instance(cls,
                          ) -> 'ROS2LaunchManager':
         return ROS2LaunchManager(app_instance=app_instance)
 
-    def read(self,
-             filename: str,
-             *,
-             package: Optional[str] = None,
+    def read(self, filename: str,
              argv: Optional[Sequence[str]] = None
              ) -> LaunchConfig:
-        """Produces a summary of the effects of a launch file.
-
-        Parameters
-        ----------
-        filename: str
-            The name of the launch file, or an absolute path to the launch
-            file inside the container.
-        package: str, optional
-            The name of the package to which the launch file belongs.
-        argv: Sequence[str], optional
-            An optional sequence of command-line arguments that should be
-            supplied to :code:`roslaunch`.
-
-        Raises
-        ------
-        PackageNotFound
-            If the given package could not be found.
-        LaunchFileNotFound
-            If the given launch file could not be found in the package.
-        """
-        raise NotImplementedError
+        # TODO Implement this by constructing a launch file reader
+        pass
```

Comment:
Accidentally deleted docstrings.

Reply:
Also, prefer `raise NotImplementedError` to `pass` as it avoids allowing the software to continue in a bad state.

Reply:
Done